src/_P088_Ping.ino

```diff
@@ -161,39 +183,29 @@ boolean Plugin_088(byte function, struct EventStruct *event, String& string)
   case PLUGIN_INIT:
   {
     initPluginTaskData(event->TaskIndex, new P088_data_struct());
-    if (!P088_data) {
-      P088_data = new P088_icmp_pcb();
-      P088_data->m_IcmpPCB = raw_new(IP_PROTO_ICMP);
-      raw_recv(P088_data->m_IcmpPCB, PingReceiver, NULL);
-      raw_bind(P088_data->m_IcmpPCB, IP_ADDR_ANY);
-      P088_data->instances = 1;
-    } else {
-      P088_data->instances++;
-    }
     UserVar[event->BaseVarIndex] = 0;
+    addLog(LOG_LEVEL_ERROR, "PLUGIN_INIT r " + String(P088_data->instances));
```

Comment:
This is not using the F() macro for the string.
```c++
if (logLevelActiveFor(LOG_LEVEL_ERROR)) {
  String log = F("PLUGIN_INIT r ");
  log += P088_data->instances; 
  addLog(LOG_LEVEL_ERROR, log);
}
```

Reply:
I added this log by mistake, during debug, removing